### ISSUE TITLE
Taiga 1.4.0 are broken on windows 11.

### ISSUE THREAD

**Issue by MeGaNeKoS** [erengy/taiga#1106]
I cant update the anime relation, or manually update the file in windows 11. My pc run on windows 11 22H2. The log only show an error in http.
```
2023-01-07 04:34:29 [Error] http.cpp:144 taiga::http::detail::SendRequest | Failure when receiving data from the peer (56)
```

**Comment by MeGaNeKoS** [comment]
P.S. I've try to manually add the entries for bungou stray dog 4th season but it still doesnt recognized. After replace the file manually, then it work detecting it. But the auto update still not working

**Comment by erengy** [comment]
It's most likely due to the bug in libcurl that was recently fixed in b6f440e48b2827dd9b0c680861cf6c498748c306. Should be fixed in the [latest build](https://taiga.moe/latest.html).

**Comment by MeGaNeKoS** [comment]
Ok. the problem fixed in the lates build. Waiting for the patch released.
